src/pages/UpdateEventPage/UpdateEventPage.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { formatISO } from 'date-fns';
+
+import FailedFetchAlert from 'src/components/FailedFetchAlert';
+import LoadableContent from 'src/components/LoadableContent';
+import Loader from 'src/components/Loader';
+import { UpdateEventWrapper } from './styled';
+import { fetchEvent, updateEvent } from 'src/api/events';
+import EventForm from 'src/components/EventForm';
+import { EventDataOutput } from 'src/components/EventForm/EventForm';
+
+type UpdateEventPageProps = {
+  eventId: string;
+  '*': string;
+};
+
+function UpdateEvent({
+  navigate,
+  eventId,
+  ...rest
+}: UpdateEventPageProps & RouteComponentProps) {
+  const tmpEventName = rest['*'];
+
+  const handleSubmit = async (eventData: EventDataOutput) => {
+    await updateEvent(
+      eventId,
+      formatISO(eventData.startTime),
+      formatISO(eventData.endTime),
+      eventData.name,
+      eventData.description,
+      eventData.eventType,
+      eventData.imageUrl,
+      eventData.imageAlt,
+      eventData.location
+    );
+    navigate!(`/event/${eventId}/${eventData.name}`);
```

Comment:
What if `updateEvent` throws an exception? The user won't be able to see the error message displayed be EventForm.